src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/BaseQualityRankSumTestUnitTest.java

```diff
@@ -75,35 +75,6 @@ public void testBaseQual() {
     }
 
     @Test
-    public void testBaseQualRawAnnotate() {
```

Comment:
Why was this test case deleted?

Reply:
I moved it to AS_BaseQualityRankSumUnitTest and updated it to be consistent with the new behavior